src/components/VolunteerApprovalGrid/Row/Row.module.css

```diff
@@ -73,7 +80,7 @@
 
 .Detail {
   display: flex;
-  width: 300px;
+  width: 280px;
```

Comment:
why the update?

Reply:
300px on this would push the other section off the row. Even with this change, there are certain cases where an especially long email will hang off the row